docs/reference/configure-fcl.mdx

```diff
@@ -2,31 +2,76 @@
 title: How to Configure FCL
 ---
 
-## Configuration
+# Configuration
 
-FCL has a mechanism that lets you configure various aspects of FCL.
-The main idea here (from an FCL perspective) should be that when you move from one instance of the Flow Blockchain to another (Local Emulator to Testnet to Mainnet) the only thing you should need to change (once again from an FCL perspective) is your configuration.
+FCL has a mechanism that lets you configure various aspects of FCL. The main idea is that you can move from one instance of the 
+Flow Blockchain to another, for example, the `Local Emulator`, `Testnet`, or `Mainnet`. So the only thing you need to change is your configuration.
 
 ## Setting Configuration Values
 
-Values only need to be set once. We recomend doing this once and as early in the life cycle as possible.
-To set a configuation value, the `put` method on the `config` instance needs to be called, the `put` method returns the `config` instance so they can be chained.
+First, you need to call the `put` method on the `config` instance to set a configuration value because the put method will return the `config` instance so it can be chained.
+We recommend doing this once and as early in the life cycle.
+
+> Values only need to be set once.
+
+## Example 1
```

Comment:
all of the titles should convey what the goal of the example is. In this case, setting the configuration

Reply:
I already stated it before the example` Setting Configuration Values`, then below each example, I also made it clear what configuration setting it represents.

## Example 1
Local emulator configuration

I think it will be wordy adding the `setting the configuration` again.